pokemongo_bot/cell_workers/transfer_pokemon.py

```diff
@@ -153,7 +153,9 @@ def release_pokemon(self, pokemon):
             data={
                 'pokemon': pokemon.name,
                 'cp': pokemon.cp,
-                'iv': pokemon.iv
+                'iv': pokemon.iv,
+                'ncp': '?',
```

Comment:
You don't gain xp when exchanging pokemons. After we may have to choose what stats are the most relevant to display. I am displaying what I think people are interested to see to make sure they don't release good pokemons.